<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
	xmlns:context="http://www.springframework.org/schema/context"
	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd"
    default-lazy-init="true" default-autowire="byName">

	<context:component-scan base-package="net._01001111.openidtester" />
	
	<import resource="applicationContext.xml" />
	
    <bean class="org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping" />

	<bean class="org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter" />

	<bean class="org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter" />

	<bean id="filenameController" class="net._01001111.openidtester.webapp.controller.UrlFilenameViewController" />
	
	<bean id="urlMapping"
		class="org.springframework.web.servlet.handler.SimpleUrlHandlerMapping">
		<property name="mappings">
			<value>
				/=filenameController
			</value>
		</property>
		<property name="order" value="2" />
	</bean>
	
	<bean id="exceptionResolver" class="org.springframework.web.servlet.handler.SimpleMappingExceptionResolver">
		<property name="exceptionMappings">
			<map>
				<entry key="javax.servlet.ServletException" value="error" />
			</map>
		</property>
		<property name="defaultErrorView" value="error" />
		<property name="warnLogCategory" value="net._01001111.openidtester" />
	</bean>
</beans>